x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/Driver.java

```diff
@@ -146,15 +176,19 @@ private SubscribableListener<Void> run(TimeValue maxTime, int maxIterations) {
             }
```

Comment:
Shouldn't we also update the cpu time if we go over the max iterations above this line?

Reply:
Oh there's a few problems here! I think I need to `+=` the CPU time rather than `=`. Let me write some tests for it.

Reply:
I've pushed an update here.